### ISSUE TITLE
Dropout rate should be 0.1

### ISSUE THREAD

**Issue by mdering** [alexis-jacq/Pytorch-Sketch-RNN#5]
The original implementation calls for a 0.9 keep probability but yours calls for a 0.9 dropout probability.

**Comment by varshaneya** [comment]
Yes, that is true. @alexis-jacq  You need to change that in the class of Hyperparameters.

**Comment by hartsantler** [comment]
I do not think a dropout rate of 0.1 is optimal.  I forked from varshaneya, where he had set dropout rate to 0.1.  But then i bumped the dropout rate to 0.4, and get much better results.  Note that this is on a 64x64 network, and i skip every other point in each stroke from the training data set.
https://youtu.be/3aphVXrMXuI

**Comment by leejseo** [comment]
@alexis-jacq Or you may just accept PR #9 .
